### ISSUE TITLE
Changing state of shower doesn't turn on Touch screen keypad

### ISSUE THREAD

**Issue by 38Cherry** [niemyjski/homeassistant-kohler#18]
Using the HA action to change the state of the shower from off to on doesn't turn on the Touchscreen in the shower.  I verified that using Alexa through Kohler Konnect does turn on the screen so there is a way to do it.  I haven't been able to isolate the api call but i'm sure it's there.

**Comment by niemyjski** [comment]
I haven't seen anything in the api todo it. If you find it, we'll fix it.

**Comment by dcmeglio** [comment]
Kohler Konnect uses a totally different approach that requires the cloud, the local device API doesn't offer this ability.

**Comment by niemyjski** [comment]
Can we use a hybrid approach? Use cloud if network capabilities are there? Fallback to local? Mine isn't connected via HomeKit/cloud

**Comment by dcmeglio** [comment]
It’s possible. I did something similar in my Hubitat integration. The Azure IoT is a bit of a pain to work with though. I’m still working on learning HA programming and python so it’s taking me a bit to get up to speed

**Comment by niemyjski** [comment]
Yeah, I almost think we need to update the underlying library to support both cloud and local and use local where possible / makes sense. I know all my mirrors and facets are cloud only (minus homekit). I haven't yet upgraded to the siri/homekit shower component (if you have this, let me know what you think).

**Comment by niemyjski** [comment]
> I've investigated the issue (#18) regarding turning on the DTV+ touchscreen keypad when the shower state changes.
>
>Based on my research into the local CGI API exposed by the Kohler DTV+ controller (which the kohler-python library uses):
>
>The quick_shower.cgi API call used by Home Assistant to turn the shower on successfully starts the water but does not trigger a screen wake event on the physical keypad.
There are no other documented local CGI endpoints (like a wake_screen.cgi) available on port 80 that we can call to manually wake up the screen.
